gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
         <NxH3>Navigating the Gallery</NxH3>
       </NxTile.SubsectionHeader>
       <NxP>
-        The sidebar at left can be used to navigate to the various examples, which are organized into groups
-        as follows:
+        The sidebar at left can be used to navigate to the various components, styles and utilities. The navigation
+        is grouped by general purpose, as follows:
       </NxP>
-      <NxH4>React Components</NxH4>
+      <NxDescriptionList>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Alerts and Indicators</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes documentation regarding components which are intended to convey specific types of
+            information to the user. For example, alert components for informing the user of events, and counters, tags,
+            and threat indicators for creating consistent ways of present particular aspects of a data entity.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Buttons and Dropdowns</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes components that the user would activate in order to initiate some action – buttons,
+            dropdowns and other menus, and also links.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Forms</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This sections includes all components that are typically used within a form. Components for forms themselves,
+            form structures such as fieldset, and for various kinds of form fields all fall within this section.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Data Presentation</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section groups together components which are intended for organizing and displaying data. This includes
+            views such as lists, tables, and trees, and also visualizations such as the donut chart.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Layout</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Components which make up the "page skeleton" are included in this section. This includes page-level components
```

Comment:
This section includes documentation on how to use different page components that determine the look and feel and the general navigation, in a consistent manner.